src/modules/newDeploy.ts

```diff
@@ -0,0 +1,88 @@
+import chalk from 'chalk'
+import { Messages } from '../lib/constants/Messages'
+import {
+  createRegistryClient,
+  parseLocator,
+  logger,
+  ManifestEditor,
+  ManifestValidator,
+  promptConfirm,
+  SessionManager,
+  returnToPreviousAccount,
+  switchAccount,
+} from 'vtex'
+
+const switchToVendorMessage = (vendor: string): string => {
+  return `You are trying to deploy this app in an account that differs from the indicated vendor. Do you want to deploy in account ${chalk.blue(
+    vendor
+  )}?`
+}
+
+const promptDeploy = (app: string) => promptConfirm(`Are you sure you want to deploy app ${app}`)
+
+const deployRelease = async (app: string, force: boolean): Promise<boolean> => {
+  const { vendor, name, version } = parseLocator(app)
```

Comment:
I would call the variable `name` of `appName`.

Reply:
If you call `parseLocator` and get `appName` will return `undefined`

Reply:
Huum I think that we would have to change the `parseLocator` (and possibly even other places) so let this that way!

Reply:
Ok!